parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
I just followed the general pattern used in ctags. If you search for "static vString", you'll find lots of them in the code. Yes, it's not good if you want to make it concurrent but there's really a LOT of work if you want to make the code thread-safe (and personally I'd rather go the multi-process way than the multi-threaded way).

How do you want to achieve this using stack allocation?

As I said, I can drop this patch, I don't care much but it doesn't solve anything - code like this is everywhere.